### DISCUSSION TITLE
Support for multiple package installation for Red Hat derivitives.

### DISCUSSION THREAD

**Discussion by bschonec** [vmware/packer-examples-for-vsphere#514] [Help] [answered]
I have a need to install multiple packages in the %packages section of the kickstart configuration but other than hard-coding these values, I don't see a way to support such customization.

My gold images need sudo, open-vm-tools and perl RPM packages installed during the creating of the RHEL template.

The same problem exhibits itself when I want to customize logical volume management.  Do I really need to create a new HCL template for every variation of a "gold" image?

I haven't found much if anything regarding support for loops or lists (arrays) in templates.  Wouldn't a simple loop or something be appropriate for package installation (at least)?

**Comment by tenthirtyam** [comment, marked as answer]
> _The same problem exhibits itself when I want to customize logical volume management._

This is planned for a future update to this sample project. A pull request is already present and awaiting my testing when time permits. A similar approach could be taken for packages.

**Comment by tenthirtyam** [comment]
- https://github.com/vmware-samples/packer-examples-for-vsphere/issues/694
